CITATION.cff

```diff
@@ -0,0 +1,68 @@
+cff-version: 1.2.0
+abstract: "The Core Imaging Library (CIL) is an open-source Python framework for tomographic imaging 
+           with particular emphasis on reconstruction of challenging datasets. Conventional filtered 
+           backprojection reconstruction tends to be insufficient for highly noisy, incomplete, non-standard
+           or multichannel data arising for example in dynamic, spectral and in situ tomography. CIL 
+           provides an extensive modular optimization framework for prototyping reconstruction methods 
+           including sparsity and total variation regularization, as well as tools for loading, preprocessing 
+           and visualizing tomographic data."
```

Comment:
Sorry I am very fussy, but as we use 's' rather than 'z' in most of these words in the software e.g. optimi`s`ation, I think it would make sense to be consistent with that in the docs. Here we also have regulariz/sation and visualiz/sing

Reply:
sure, this piece of text is from https://github.com/TomographicImaging/CIL/blob/0cc8cb101578c101596f0b6f6d7685659ab715db/README.md?plain=1#L8